data/common.yaml.tmpl

```diff
@@ -485,9 +485,11 @@ ceph_mon_secret: {{ config.ceph_mon_secret }}
 {% if config.ceph_osd_devices %}
 ceph_osd_devices:
 {% if config.ceph_osd_devices is mapping %}
-{% for disk in config.ceph_osd_devices %} {{ disk }}:{% if disk.journal is defined %}
-    {{ disk.journal }}
-{% endif %}
+{% for disk, values in config.ceph_osd_devices.iteritems() %}
```

Comment:
you should sanitize `journal` and `journalsize` keys (that are parameters in puppet-ceph [1]) because otherwise someone can define a key (parameter) that does not exist in puppet-ceph. Also, keep in mind these parameters are optional so we don't need them to build common.yaml.

[1] https://github.com/enovance/puppet-ceph/blob/master/manifests/osd/device.pp#L20-L21